app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  before_action :find_post, only: [:create]
+
+  def create
```

Comment:
- It's a good practice to always define the `new` method with the `create` method. The new method is used by ActiveRecord to build the respective object for the controller.